vcbuild.bat

```diff
@@ -128,12 +121,6 @@ echo Failed to find Visual Studio installation.
 goto exit
 
 :msbuild-found
-
-:project-gen
-@rem Skip project generation if requested.
-if defined noprojgen goto msbuild
-
-@rem Generate the VS project.
 python configure %download_arg% %i18n_arg% %debug_arg% %snapshot_arg% %noetw_arg% %noperfctr_arg% --dest-cpu=%target_arch% --tag=%TAG%
```

Comment:
These lines should not be deleted here. They should be deleted from the other copy above.